mlflow/ml-package-versions.yml

```diff
@@ -620,7 +620,7 @@ transformers:
       "< 4.30.0": ["tensorflow<2.13.0"]
       # Installing `transformers==4.34.1` with `datasets` results in installing `huggingface_hub==0.17.3`.
       # `accelerate>=0.32.0` is incompatible with `huggingface_hub==0.17.3`.
-      "== 4.34.1": ["accelerate<0.32.0"]
+      "== 4.34.1": ["accelerate<0.32.0", "sentence-transformers<3.1.0"]
```

Comment:
Installing the latest version of sentence-transformers results in very old `setfit` being installed.

```
Collecting setfit
  Downloading setfit-1.0.2-py3-none-any.whl.metadata (11 kB)
INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. See https://pip.pypa.io/warnings/backtracking for guidance. If you want to abort this run, press Ctrl + C.
Collecting evaluate>=0.3.0 (from setfit)
  Using cached evaluate-0.4.3-py3-none-any.whl.metadata (9.2 kB)
  Downloading evaluate-0.4.2-py3-none-any.whl.metadata (9.3 kB)
  Downloading evaluate-0.4.1-py3-none-any.whl.metadata (9.4 kB)
Collecting responses<0.19 (from evaluate>=0.3.0->setfit)
  Downloading responses-0.18.0-py3-none-any.whl.metadata (29 kB)
Collecting evaluate>=0.3.0 (from setfit)
  Downloading evaluate-0.4.0-py3-none-any.whl.metadata (9.4 kB)
  Downloading evaluate-0.3.0-py3-none-any.whl.metadata (9.1 kB)
Collecting setfit
  Downloading setfit-1.0.1-py3-none-any.whl.metadata (11 kB)
  Downloading setfit-1.0.0-py3-none-any.whl.metadata (11 kB)
  Downloading setfit-0.7.0-py3-none-any.whl.metadata (19 kB)
  Downloading setfit-0.6.0-py3-none-any.whl.metadata (19 kB)
  Downloading setfit-0.5.0-py3-none-any.whl.metadata (16 kB)
  Downloading setfit-0.4.1-py3-none-any.whl.metadata (15 kB)
```

```
...
setfit                                   0.4.1        2022-11-08 
...
```